resources/liquibase/db.changelog.xml

```diff
@@ -0,0 +1,72 @@
+   <changeSet id="202302020908" author="moroti">
+     <createTable tableName="users"
+             schemaName="public"
+             >
+         <column name="id" type="INTEGER" autoIncrement="true">
+            <constraints nullable="false"
+               primaryKey="true"
+               primaryKeyName="users_pkey"
+            />
+         </column>
+         <column name="email" type="VARCHAR(255)">
+            <constraints nullable="false"/>
+        </column>
+        <column name="password" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="role" type="VARCHAR(10)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="firstname" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="lastname" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="gender" type="VARCHAR(10)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="dob" type="date">
+                <constraints nullable="false"/>
+            </column>
+            <column defaultValueBoolean="false" name="isActive" type="BOOLEAN"/>
+            <column defaultValueBoolean="false" name="isVerified" type="BOOLEAN"/>
+            <column name="createdAt" type="TIMESTAMP WITH TIME ZONE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="updatedAt" type="TIMESTAMP WITH TIME ZONE">
+                <constraints nullable="false"/>
+            </column>
+     </createTable>
+   </changeSet>
+   <changeSet author="moroti" id="202302021026">
+        <createTable tableName="books">
+            <column autoIncrement="true" name="id" startWith="3" type="INTEGER">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="books_pkey"/>
+            </column>
+            <column name="title" type="VARCHAR(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="isbn" type="VARCHAR(10)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="author" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column defaultValueBoolean="true" name="isAvailable" type="BOOLEAN">
+                <constraints nullable="false"/>
+            </column>
+            <column name="createdAt" type="TIMESTAMP WITH TIME ZONE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="updatedAt" type="TIMESTAMP WITH TIME ZONE">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+     <changeSet author="moroti" id="202302021029">
```

Comment:
Include a `createdBy` column and `lastUpdatedBy` column to keep track of the admin that added the record and the last person that updated the record.